js/utils.js

```diff
@@ -1588,6 +1588,20 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
               }
             });
             break;
+
+          case 'link-query-parameter':
+            item.value = Fliplet.Navigate.query[item.fieldValue] || '';
```

Comment:
It looks sensible to have a default, but because `Fliplet.Navigate.query` is always an object, the worst case scenarios is that it'll be `undefined`, which might be OK.

Please tell us was there a reason or specification that made you use `''` as the default value? Did we specify it somewhere?

Reply:
@tonytlwu you are right the check is not needed, removed the unnecessary code.